Formula/pcp.rb

```diff
@@ -0,0 +1,19 @@
+class Pcp < Formula
+  desc "📦 Command-line peer-to-peer data transfer tool based on libp2p"
+  homepage "https://github.com/dennis-tra/pcp"
+  url "https://github.com/dennis-tra/pcp", using: :git, tag: "v0.4.0"
```

Comment:
```suggestion
  url "https://github.com/dennis-tra/pcp.git",
      tag:      "v0.4.0",
      revision: "7f638fe42f6dbd17e5bf5a7be5854220e2858eb2"
```
Let's try to be consistent with other formulae here. Note the alignment of the arguments to `url`.

Reply:
ah okay, that's also how it was formatted in the documentation.